extensions/ql-vscode/package.json

```diff
@@ -92,7 +92,7 @@
     },
     "jsonValidation": [
       {
-        "fileMatch": "workspace-databases.json",
+        "fileMatch": "databases.json",
```

Comment:
I think this is too generic now. It means that if someone has our extension installed, and they open any databases.json file, they'd get JSON validation applied with our schema.

We should look at ways to make it more specific (e.g. by looking at the file path a bit more).